src/pages/Register.vue

```diff
@@ -186,12 +186,12 @@ export default {
           email: this.mail,
           authenticated: true
         }
+        this.$router.go(-2) // Assumes that user came via /login page
```

Comment:
Okay, I go with one page back.
What is here the reason for - 2

Reply:
Here, the same scenario as above applies, except that the user has no account yet. Therefore, the user goes to the `/login` page and then to the `/register` page. When the user has successfully registered, we go back 2 pages instead of 1 to skip the `/login` page.

Reply:
Okay, I see it's already merged. Let's wait, what the UX-Designer has to offer.